README.rst

```diff
@@ -78,6 +78,28 @@ Want to know if a word you're proposing exists in codespell already? It is possi
     echo "word" | codespell -
     echo "1stword,2ndword" | codespell -
 
+Using a config file
+-------------------
+
+Command line options can also be specific in a config file.
+
+When running ``codespell``, it will check in the current directory for a file
+named ``setup.cfg`` or ``.codespellrc`` (or a file specified via ``--config``),
+containing an entry named ``[tool:codespell]``. Each command line argument can
+also be specified in this file, for example::
+
+    [tool:codespell]
```

Comment:
Tool is the new standard then? I see flake8 doesn't have a prefix.

Reply:
There is no standard. Each tool does what it wants. Some use the tool prefix (pytest), some do not. We could take it out. It's a matter of bike-shedding.